apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/controller/OnboardingController.java

```diff
@@ -336,9 +336,10 @@ public Uni<Response> update(@PathParam(value = "onboardingId") String onboarding
     }
 
     @Operation(summary = "In the addition administrator flow, it checks " +
-            "if the new manager is equal from old one, returning true ")
+            "if the new manager is equal to the old one, returning true or false")
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
     @Path("/check-manager")
```

Comment:
it might be better to use nouns in the endpoint paths, not verbs. [The endpoints should use nouns, signifying what each of them does](https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/). For ex. **/onboardings/manager/validation** or you could create an endpoint **/onboardings/manager/{id}** that return a manager, in case of it exists your check will be true on your bff....